src/components/look-controls.js

```diff
@@ -56,9 +59,19 @@ module.exports.Component = registerComponent('look-controls', {
 
   tick: function (t) {
     var data = this.data;
+    var hasPositionalTracking;
+
     if (!data.enabled) { return; }
+
     this.controls.standing = data.standing;
-    this.controls.userHeight = this.getUserHeight();
+
+    hasPositionalTracking = this.hasPositionalTracking !== undefined
+      ? this.hasPositionalTracking
+      : checkHasPositionalTracking();
+    if (hasPositionalTracking) {
```

Comment:
Trying to understand. How come we still want to  apply user height if there is positional tracking? Seems to me you don't want to apply an offset if your height is tracked, or we should check more specifically that there are no stage parameters? And how come it was adding up more and more each time enter VR?